### ISSUE TITLE
Oscilloscope

### ISSUE THREAD

**Issue by ME-TECH-ELECTRONICS** [wokwi/wokwi-features#640]
Every time I start the simulation I get a problem like whether the microcontroller is giving a output signal or not. When I tried other simulation software I got the oscilloscope feature my would like to get this feature to added in wokwi

**Comment by urish** [comment]
Hello! Have you tried the [logic analyzer](https://docs.wokwi.com/guides/logic-analyzer)?

**Comment by ME-TECH-ELECTRONICS** [comment]
Yes I tried but it requires another piece of software to view the signal. But a oscilloscope i mentioned here can show the realtime signal in wokwi without any other software.

**Comment by urish** [comment]
Something like this? 

* https://wokwi.com/projects/360935045930911745
* https://wokwi.com/projects/361237697368753153

**Comment by drf5n** [comment]
The Wokwi-Chip-Scope is developed and documented at:

https://github.com/Dlloydev/Wokwi-Chip-Scope
